### DISCUSSION TITLE
What is Stratum and Strata?

### DISCUSSION THREAD

**Discussion by himby** [TerriaJS/terriajs#6102] [Q&A] [answered]
I'm in the process of migrating from V7 to V8, and it feels like there is quite a steep learning curve to understand the new architecture. 
What is a Strata and a Stratum? And what is this CommonStrata enum used for?  The documentation talks about them like they are wide known concepts in software terminology, but I've never heard of it. And google doesn't seem to either. Would be really appreciated with some more details in the documentation regarding this.

I've read through:
[New Model Layer](https://docs.terria.io/guide/contributing/model-layer/)
and:
[Architecture](https://docs.terria.io/guide/contributing/architecture/) 

Thanks for all your hard work anyway ! :)

**Comment by zoran995** [comment, marked as answer]
You can check #5873 for more details.

 Also, there is an old description @kring wrote in #2959, I believe he invented the term strata. 
> To that end, I've been experimenting with some significant changes to the Model layer, such as switching from Knockout to MobX, changing the way we do many operations (like adding things to the map) so make them far less side effect ridden, using TypeScript, and separating model properties into separate layers (**I call them strata, cause the term layer is already used in mapping**).

Hopefully, someone from the team can give you more details if needed

**Reply by himby** [reply]
Thanks a lot @zoran995 ! That was helpful 👍
